### PR TITLE
Video games filtered with respect to video duration.

### PR DESCRIPTION
**Pull request by bhargaviks**
Hi, 
This PR consists of filtering the Active videos wrt their duration. The filtering has been done in this way:
1. [Short video - less than 15 minutes](https://screenshot.googleplex.com/bd5d7cdb-da2b-4f02-b0ad-1ccb05860dbc.png)
2. [Medium video - less than 30 minutes and greater than 15 minutes.](https://screenshot.googleplex.com/6dacaad4-6759-44cf-adb3-fb529b88a6f9.png)
3. [Large videos - more than 30 minutes. ](https://screenshot.googleplex.com/ee04c64d-202d-46f2-ae6b-9909712a7990.png)
I have deployed the recent changes, so the deployed version should contain these filter changes. 
This is how the [drop down now looks](https://screenshot.googleplex.com/aa253ad9-d800-4a5e-9311-922aa732bac5.png)
These three options in the drop down menu are associated with the same field on linkFilters.videoLength, and these are edited when the length changes, and are deleted when the user clicks any other dropdown menu that are not related to the length of the video. Please take a look at it!

### CODE REVIEW

File: src/main/app/src/hooks/browseActivitiesFilter.js [hunk 1]
```diff
@@ -27,25 +27,41 @@ const filterActivities = (links, activityType, activityTypes, linkFilters, updat
...
 }
 
-/** Filter list of videos. */
+/** 
+  Filter list of videos.
+  Short videos duration  -  [0,900]
+  Medium length videos - [901,1800]
+  Large length videos- (>=1801)
+*/
 const filterVideos = (video, filters) => {
```
Comment (mbouadeus) [inline src/main/app/src/hooks/browseActivitiesFilter.js:73]: The filterVideo function doesn't filter by video category with this change. Wouldn't adding a check for what type of filter it is and then filtering either by category or by duration then be better?

**Review by mbouadeus — APPROVED** [review]

File: src/main/app/src/hooks/browseActivitiesFilter.js (hunk 1 above)
Comment (bhargaviks) [reply src/main/app/src/hooks/browseActivitiesFilter.js:73]: Yes for sure. I wanted to add that when Trinity adds the categories filter inside this.

File: src/main/app/src/hooks/browseActivitiesFilter.js [hunk 2]
```diff
@@ -27,25 +27,41 @@ const filterActivities = (links, activityType, activityTypes, linkFilters, updat
...
-    return (min === 0 && max === 0) ||                 // If there are no restrictions on the number of players
-        (min === 0 && max >= input) ||              // If no restriciton on minPlayer, but there is a max limit satisfied
-        (max === 0 && min <= input) ||              // If no restriction on maxPLayer, but minimum limit is satisfied
-        (min <= input && max >= input);         // If both are non-zero, and the game players are within limit. 
-    
-    // Otherwise don't return. 
+  var min = parseInt(game.minPlayer,10);
+  var max = parseInt(game.maxPlayer,10);
+  
+  return (min === 0 && max === 0) ||                 // If there are no restrictions on the number of players
```
Comment (isaacwdavis) [inline src/main/app/src/hooks/browseActivitiesFilter.js:33]: nit: inconsistent spacing between comment and code

**Review by pengxianzhe — APPROVED** [review]

File: src/main/app/src/pages/BrowsePage.js [hunk 3]
```diff
@@ -44,7 +47,15 @@ const BrowsePage = ({ links, activityType, updateActivityType, updateActivity, u
           updateLinksFilters(Object.assign(linkFilters, {numOfPlayers: value}));
           break;
         case activityTypes.VIDEOS:
-          updateLinksFilters(Object.assign(linkFilters, {videoType: value}));
+          // If the activity length is the filter that needs to be taken care of. 
+          // These three drop down options come together as one in videoLength inside linkFilters. 
+          if(value==="short" || value === "medium" || value==="large") {
```
Comment (isaacwdavis) [inline src/main/app/src/pages/BrowsePage.js]: nit: inconsistent spacing

File: src/main/app/src/hooks/browseActivitiesFilter.js (hunk 2 above)
Comment (isaacwdavis) [inline src/main/app/src/hooks/browseActivitiesFilter.js]: nit: inconsistent spacing between code and comments

File: src/main/app/src/pages/BrowsePage.js [hunk 4]
```diff
@@ -7,6 +7,7 @@ import VideoCard from '../constants/VideoCard.js';
 import filterActivities from '../hooks/browseActivitiesFilter.js';
 import {GamesFilterBar, ArticlesFilterBar, VideosFilterBar} from '../constants/ActivitiesFilterBar';
 
+
```
Comment (pengxianzhe) [inline src/main/app/src/pages/BrowsePage.js]: nit: Why do you need an extra empty line here?

**Review by pengxianzhe — APPROVED** [review]

**Review by isaacwdavis — APPROVED** [review]

**Comment by bhargaviks** [comment]
Fixed the feedback! Thank you!

**Pull request merged**